deeprank2/utils/buildgraph.py

```diff
@@ -318,4 +318,4 @@ def get_surrounding_residues(structure: Union[Chain, PDBStructure], residue: Res
 
             close_residues.add(structure_atom.residue)
 
-    return close_residues
+    return list(close_residues)
```

Comment:
`get_surrounding_residues` misses the type hint about the returned variable

Reply:
Returning a list instead of a set, in the atomic resolution build case may cause an issue if (and only if) `variant_residue` is not unique (is this a possible scenario?), and is appended to the list `residues` again. It's also true that `atoms` is then defined as a set and populated according to `residues`, so even if the latter scenario is possible, I guess that defining then `atoms` as a set could avoid the issue. Maybe you've already carefully thought about that and we've already reached a conclusion, but just in case here's my comment. Let me know if I need to double check this more carefully.

Reply:
The way I remember it, but would need to double check, is that a set was returned here, which was immediately converted to a list by the method that uses this. I short-cut that by converting to list before returning.
The logic didn't change. atoms are added to a set, so that duplicates are culled. That set is then converted to a list for usage by the code.

Reply:
also, see my comment above.

Reply:
> `get_surrounding_residues` misses the type hint about the returned variable

fixed in f42752a4